docker/truffle/Dockerfile

```diff
@@ -7,5 +7,4 @@ COPY docker/truffle/run.sh /app/run.sh
 
 # Dependencies will Install app dependencies
 RUN apk add --update --no-cache --virtual build-dependencies git python make g++ ca-certificates
-RUN npm install --production
```

Comment:
I will put this into a separate PR, as it is an fix

Reply:
Check #280 for a fix.